server/tournaments/index.ts

```diff
@@ -2172,6 +2220,73 @@ const commands: Chat.ChatCommands = {
 					return this.sendReply(`Usage: ${this.cmdToken}${this.fullCmd} <on|off>`);
 				}
 			},
+			recenttours(target, room, user) {
+				room = this.requireRoom();
+				this.checkCan('declare', null, room);
+				const num = parseInt(target);
+				if (!target ||
+					(!this.meansNo(target) && toID(target) !== 'forcedelete' &&
+						(isNaN(num) || num > 15 || num < 0))) {
+					return this.parse(`/help tour settings`);
+				}
+				if (!room.settings.tournaments) room.settings.tournaments = {};
+				if (!isNaN(num) && num <= 15 && num >= 1) {
+					if (room.settings.tournaments.recentToursLength === num) {
+						throw new Chat.ErrorMessage(`Recent tournament threshold is already set to ${num}.`);
+					}
+					room.settings.tournaments.recentToursLength = num;
+					room.saveSettings();
+					this.privateModAction(`Recent tournament threshold was set to ${num} by ${user.name}.`);
```

Comment:
This wording is a little clunky; maybe "number of recent tournaments to record"?